lib/index.js

```diff
@@ -299,7 +299,7 @@ Server.prototype = {
         });
       }
     }, installLogger));
-    this.pluginLoader.loadPlugins();
+    yield this.pluginLoader.loadPlugins();
```

Comment:
Thanks good comment. I believe it works because it's a legacy generator function, basically adding it with an async method that has a promise chain and eventually gets to a then() closes the promise chain, which is basically like closing the last then() of a yield, which is why code continues after the yield and it works
We can see this again with something like, https://github.com/zowe/zlux-server-framework/pull/243/files/1dd87148bb1f85271ad54beb2c51b73adfdb3eed#diff-6d186b954a58d5bb740f73d84fe39073R334 that points to another method, that's not a generator but houses a promise chain